File: R/tests/testthat/test_banditNarm_lapse.R
context("Test banditNarm_lapse")
library(hBayesDM)

test_that("Test banditNarm_lapse", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(banditNarm_lapse(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})

File: Python/tests/test_banditNarm_lapse.py
import pytest

from hbayesdm.models import banditNarm_lapse


def test_banditNarm_lapse():
    _ = banditNarm_lapse(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()
